TestApp/ios/Pods/Target Support Files/Pods-TestApp/Pods-TestApp-acknowledgements.plist

```diff
@@ -46,28 +46,6 @@ THE SOFTWARE.</string>
 			<key>Type</key>
 			<string>PSGroupSpecifier</string>
 		</dict>
-		<dict>
-			<key>FooterText</key>
-			<string>Visual Studio App Center Plugin for React Native
-
-Copyright (c) Microsoft Corporation
-
-All rights reserved.
-
-MIT License
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.</string>
-			<key>License</key>
-			<string>MIT</string>
-			<key>Title</key>
-			<string>AppCenterReactNativeShared</string>
-			<key>Type</key>
-			<string>PSGroupSpecifier</string>
-		</dict>
```

Comment:
that does not look right, was a dependency removed?

Reply:
It looks like this change and the other ones in TestApp/ios/Podfile and TestApp/ios/Pods/Target Support Files/Pods-TestApp/Pods-TestApp-acknowledgements.markdown were all changed when I ran this command: `./update-link-to-appcenter.sh` I ran that command to update TestApp to use latest SDKs, I made no changes beyond that.

Reply:
The LICENSE file also got removed due to me running that './update-link-to-appcenter.sh' command